.github/workflows/unit_tests.yml

```diff
@@ -1,4 +1,4 @@
-name: pals
+name: pals-python
```

Comment:
The `-python` suffixes added here are not really needed.

From the Python viewpoint, this repo is "pals".